cmd/scollector/conf/conf.go

```diff
@@ -82,6 +82,7 @@ type SNMP struct {
 	Community string
 	Host      string
 	MIBs      []string
+	Bridge    bool
```

Comment:
We should not add another field for this. Rather add "bridge" to the list of built in collectors here:https://github.com/bosun-monitor/bosun/blob/master/cmd/scollector/collectors/snmp.go#L17